docs/spec/spec.md

```diff
@@ -9,10 +9,90 @@ These are Kubernetes resources that been introduced using Custom Resource
+| address            | Addressable | Address of the endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
+| conditions         | Conditions  | Broker conditions.                                                                           |             |
+
+##### Conditions
+
+- **Ready.** True when the Broker is provisioned and ready to accept events.
+- **Addressable.** True when the Broker has an resolved address in it's status.
+
+#### Events
+
+- BrokerReconciled
+- BrokerUpdateStatusFailed
+
+---
+
+## kind: Trigger
+
+### group: eventing.knative.dev/v1alpha1
+
+_A Trigger represents a subscriber of events with a filter for a specific
+broker._
+
+### Object Schema
+
+#### Spec
+
+| Field        | Type           | Description                                                                                                                                                                | Constraints |
+| ------------ | -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| broker       | String         | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                        |             |
+| filter       | TriggerFilter  | Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscribing to all events. |             |
+| subscriber\* | SubscriberSpec | Subscriber is the addressable that receives events from the Broker that pass the Filter.                                                                                   |             |
+
+\*: Required
+
+#### Status
+
+| Field              | Type        | Description                                                                                              | Constraints |
+| ------------------ | ----------- | -------------------------------------------------------------------------------------------------------- | ----------- |
+| observedGeneration | int64       | The 'Generation' of the Broker that was last processed by the controller.                                |             |
+| subscriberURI      | Addressable | Address of the subscribing endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
+| conditions         | Conditions  | Trigger conditions.                                                                                      |             |
+
+##### Conditions
+
+- **Ready.** True when the Trigger is provisioned and configuration is ready to
+  deliver events to the subscriber.
+- **BrokerExists.** True when the Broker can be retrieved.
```

Comment:
So does this mean I can create a Trigger for a broker that does not have a Broker? Is that what we really want? I understand you're just documenting what we have, but I'm thinking this might not be what we want.

Reply:
can be retrieved => True when the Broker exists and is ready? Retrieved is funny. What if Broker is b0rk3d?

Reply:
It should be exists and ready. It currently is not that. I will update.